sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/ManageCustomModels.java

```diff
@@ -0,0 +1,68 @@
+import com.azure.core.util.Context;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * Sample for demonstrating common custom model management operations.
+ */
+public class ManageCustomModels {
+
+    /**
+     * Main program to invoke the demo for performing operations of a custom model.
+     *
+     * @param args Unused. Arguments to the program.
+     */
+    public static void main(final String[] args) {
+        // Instantiate a client that will be used to call the service.
+        FormTrainingClient client = new FormRecognizerClientBuilder()
+            .apiKey(new AzureKeyCredential("{api_key}"))
+            .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+            .buildClient().getFormTrainingClient();
+
+        AtomicReference<String> modelId = null;
+
+        // First, we see how many custom models we have, and what our limit is
+        AccountProperties accountProperties = client.getAccountProperties();
+        System.out.printf("The account has %s custom models, and we can have at most %s custom models",
+            accountProperties.getCount(), accountProperties.getLimit());
+
+        // Next, we get a paged list of all of our custom models
+        PagedIterable<CustomFormModelInfo> customModels = client.listModels();
+        System.out.println("We have following models in the account:");
+        customModels.forEach(customFormModelInfo -> {
+            System.out.printf("Model Id: %s%n", customFormModelInfo.getModelId());
+            // get custom model info
+            modelId.set(customFormModelInfo.getModelId());
+            CustomFormModel customModel = client.getCustomModel(customFormModelInfo.getModelId());
+            System.out.printf("Model Id: %s%n", customModel.getModelId());
+            System.out.printf("Model Status: %s%n", customModel.getModelStatus());
+            System.out.printf("Created on: %s%n", customModel.getCreatedOn());
+            System.out.printf("Updated on: %s%n", customModel.getLastUpdatedOn());
+            customModel.getSubModels().forEach(customFormSubModel -> {
+                System.out.printf("Custom Model Form type: %s%n", customFormSubModel.getFormType());
+                System.out.printf("Custom Model Accuracy: %s%n", customFormSubModel.getAccuracy());
+                if (customFormSubModel.getFieldMap() != null) {
+                    customFormSubModel.getFieldMap().forEach((fieldText, customFormModelField) -> {
+                        System.out.printf("Field Text: %s%n", fieldText);
+                        System.out.printf("Field Accuracy: %s%n", customFormModelField.getAccuracy());
+                    });
+                }
+
+            });
+        });
+
+        // Delete Custom Model
+        System.out.printf("Deleted model with model Id: %s operation completed with status: %s%n", modelId.get(),
+            client.deleteModelWithResponse(modelId.get(), Context.NONE).getStatusCode());
```

Comment:
It might be nice to include a get of that model after deletion. this way you can show what error is thrown when accessing a custom model that doesn't exist, and how to check that the model is deleted.